examples/jobs/gpu_env.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#SBATCH --job-name=gpu_env
+#SBATCH --output=/data/gpu_env_%j.out
+#SBATCH --time=00:02:00
+#SBATCH --ntasks=1
+#SBATCH -p gpu
+#SBATCH --gres=gpu:1     # land on a GPU node (g1 in 25.05 config)
+# If you prefer to hard-pin the node instead, swap the line above for:
+# #SBATCH -w g1
+
+# Make CUDA visible to Slurm-launched shells (they don't inherit container ENV)
+export CUDA_HOME=/usr/local/cuda-12.6
```

Comment:
I think we should allow flexibility here: 

```suggestion
export CUDA_HOME="${CUDA_HOME:-/usr/local/cuda-12.6}"
```

Something like this. Thoughts?